<?xml version="1.0"?>
<launch>
	<node pkg="kari_localization" type="lidar2zed" name="lidar2zed" output="screen" />

		<param name="lidar_topic" value="/center/average" />
		<param name="camera_topic" value="/camera/average/tf" />
		
		<param name="parent_frame" value="/centerlaser" />


	<node pkg="tf" type="static_transform_publisher" name="link1_broadcaster" args="0 0 0 0 0 0 /centerlaser /zed0/zed_center 100" />

</launch>


